ambari-server/src/main/java/org/apache/ambari/server/configuration/Configuration.java

```diff
@@ -2261,6 +2261,15 @@
   public static final ConfigurationProperty<String> VIEWS_HTTP_CACHE_CONTROL_HEADER_VALUE = new ConfigurationProperty<>(
       "views.http.cache-control", "no-store");
 
+  /**
+   * The value that is additional classpath for the views. It will take comma separated paths. If the individual path is jar
+   * it will be included otherwise if it is a directory then all the files inside it will be included in the classpath. Directories
+   * WILL NOT BE navigated recursively
+   */
+  @Markdown(description = "Additional class path added to each Ambari View. Semi colon separated jars or directories")
```

Comment:
Classpath value separator is contradictory: description says "semi colon separated", but comment says "comma separated".

Reply:
correct to comma separated.